services/user.py

```diff
@@ -0,0 +1,72 @@
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
+from django.core.validators import validate_email
+
+from db.models import User
+
+
+def set_user_values(
+    user: User,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]